ts/client/src/accounts/group.ts

```diff
@@ -604,6 +604,27 @@ export class Group {
     return totalAmount;
   }
 
+  /**
+   *
+   * @param mintPk
+   * @returns remaining deposit limit for mint, returns null if there is no limit for bank
+   */
+  public getRemainingDepositLimitByMint(mint: PublicKey): BN | null {
+    const bank = this.getFirstBankByMint(mint);
+    const vaultBalance = this.getTokenVaultBalanceByMint(mint);
+    const isNoLimit = bank.depositLimit.isZero();
+
+    const remainingDepositLimit = !isNoLimit
+      ? bank.depositLimit.sub(vaultBalance)
```

Comment:
use bank.nativeDeposits() and not vault balance

Reply:
and then you can move this method to the bank, something like getRemainingDepositLimit